visualization/src/pcl_visualizer.cpp

```diff
@@ -521,13 +521,15 @@ void
 pcl::visualization::PCLVisualizer::spinOnce (int time, bool force_redraw)
 {
   resetStoppedFlag ();
+#if VTK_MAJOR_VERSION < 9
```

Comment:
Join the 2 together #ifs together? This is valid only if platform is apple AND vtk is 8 or lower (my interpretation)